dojo/tools/burp_graphql/parser.py

```diff
@@ -0,0 +1,196 @@
+import logging
+import json
+import re
+
+
+from dojo.models import Endpoint, Finding
+
+logger = logging.getLogger(__name__)
+
+
+class BurpGraphQLParser(object):
+
+    def get_scan_types(self):
+        return ["Burp GraphQL API"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type  # no custom label for now
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import Burp Enterprise Edition findings from the GraphQL API"
+
+    def get_findings(self, filename, test):
+        if filename:
+            tree = filename.read()
+            try:
+                data = json.loads(str(tree, 'utf-8'))
+            except:
+                data = json.loads(tree)
+
+            scan_data = data.get('Issues')
+
+            return self.create_findings(scan_data, test)
+
+    def create_findings(self, scan_data, test):
+
+        finding_data = self.parse_findings(scan_data)
+
+        items = list()
+
+        for issue in finding_data:
+            find = Finding(title=issue.get('Title'),
+                           description=issue.get('Description'),
+                           test=test,
+                           severity=issue.get('Severity'),
+                           mitigation=issue.get('Mitigation'),
+                           references=issue.get('References'),
+                           impact=issue.get('Impact'),
+                           cwe=int(issue.get('CWE')),
+                           false_p=False,
+                           duplicate=False,
+                           out_of_scope=False,
+                           mitigated=None,
```

Comment:
```suggestion
```
Better not set it instead of forcing to `None`

Reply:
Updated in latest commit (4d3ae9f)